nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/test/resources/attachment-only.eml

```diff
@@ -0,0 +1,156 @@
+Date: Fri, 29 Jul 2011 09:41:11 +0200 (CET)
+From: Siegfried GOESCHL <siegfried.goeschl@it20one.at>
+Subject: Kunde 100029   Auftrag   3600
+To: <siegfried.goeschl@it20one.at>
+Reply-To: Siegfried GOESCHL <siegfried.goeschl@it20one.at>
+Message-ID: <ADR36000000000967@it20one.at>
+MIME-Version: 1.0
+Importance: Normal
+X-Priority: 3 (Normal)
+X-Mailer: SAP Web Application Server 7.00
+Content-Type: application/pdf;
+ name="Kunde 100029   Auftrag   3600.pdf"
+Content-Transfer-Encoding: base64
+Content-Description: Kunde 100029   Auftrag   3600

```

Comment:
@joewitt  Agreed. In a previous commit I changed the junit so that we generate the email dynamically. The files have been deleted and references to them removed from the `pom.xml`